tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -64,6 +64,25 @@ def test_not_implemented(self):
         with self.assertRaises(NotImplementedError):
             not_implemented.parse()
 
+    def test_from_metadata(self):
+
+        data = load_testdata("default.py")
+
+        original = [doc.documentation for doc in
+                    extract_documentation(data, "python", "default")]
+
+        parsed_docs = [(doc.parse(), doc.marker, doc.indent, doc.range)
+                       for doc in
+                       extract_documentation(data, "python", "default")]
+
+        docstyle_definition = DocstyleDefinition.load("python", "default")
+
+        assembled_docs = [DocumentationComment.from_metadata(
+                          doc[0], docstyle_definition, doc[1], doc[2], doc[3])
+                          for doc in parsed_docs]
+
+        self.assertEqual([str(doc) for doc in assembled_docs], original)
```

Comment:
With little modification you can also test for complete `DocumentationComment` object equality! :D
(As `DocumentationComment` uses `generate_eq`)

-->
```
original = list(extract_documentation(...))
parsed_docs = [(...)
             for doc in original]

# ...
self.assertEqual(assembled_docs, original)
```